users/models.py

```diff
@@ -1,3 +1,22 @@
+from django.contrib.auth.base_user import AbstractBaseUser
 from django.db import models
 
-# Create your models here.
+from common.models import TimeStampedModel
+
+
+class Users(TimeStampedModel, AbstractBaseUser):
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
+    email = models.EmailField(max_length=70, unique=True)
+    phone_number = models.CharField(max_length=13, unique=True)
+    passport_number = models.CharField(max_length=8, unique=True)
+    identity_code = models.CharField(max_length=15, unique=True)
+    company = models.ForeignKey("self", on_delete=models.DO_NOTHING)
+    salary = models.FloatField()
+    birth_date = models.DateField()
+    residence_permit = models.TextField(max_length=255)
+
+
+class Roles(TimeStampedModel):
```

Comment:
single form(Role)

Reply:
Does it really should be "Role"  instead "Roles" as in ER diagram?

Reply:
> single form(Role)

Does it really should be "Role"  instead "Roles" as in ER diagram?